src/components/buttons.scss

```diff
@@ -0,0 +1,106 @@
+// Fonts
+
+@import 'fonts';
+
+.fontButton {
+  font-family: "Roboto", "Verdana";
+  font-weight: 300;
+  font-size: 15pt;
+  color: white;
+}
+
+.fontInfo {
+  font-family: "Roboto", "Verdana";
+  font-weight: 300;
+  font-size: 12pt;
+  color: black;
+}
+
+// Variables
+
+$button-text: #ffffff;
+$button-student: #4be24f;
+$button-teacher: #4bdbe2;
+$button-student-hover: #359e37;
+$button-teacher-hover: #3a9b94;
+$info-background: #f7f7f7;
+$info-border: #d6d6d6;
+
+// FrontPageButton
+
+.container {
+  margin: 0 auto;
+  text-align: center;
+}
+
+.transitionHover {
+  -webkit-transition: all .1s ease-in-out;
+  -moz-transition: all .1s ease-in-out;
+  -ms-transition: all .1s ease-in-out;
+  transition: all .1s ease-in-out;
```

Comment:
Vil ikke [autoprefixer](https://www.npmjs.com/package/autoprefixer) lage ms-, moz- og webkit-regler for oss? Slik at det da holder med kun `transition: all .1s ease-in-out;`